tests/test_buffer_block.py

```diff
@@ -130,3 +130,23 @@ def test_emit_command_groups(self, patched_job):
         self.assert_num_signals_notified(6, block)
         block.stop()
         self.assertFalse(patched_job.call_count)
+
+    def test_buffer_start_signal(self):
+        event = Event()
+        block = EventBuffer(event)
+        block._backup = MagicMock()
+        self.configure_block(block, {
+            "interval": {
+                "milliseconds": 200
+            },
+            "signal_start": True
+        })
+        block.start()
+        self.assert_num_signals_notified(0, block)
+        event.wait(.1)
+        block.process_signals([Signal()])
+        event.wait(.1) # 200 miliseconds have now passed but the block should have only been active for 100
```

Comment:
I like your comment here, but you should assert what the comment says with `self.assert_num_signals_notified(0, block)`